### PR TITLE
New example of speech stream recognition with chosen model.

### PR DESCRIPTION
**Pull request by hkomine**
I create new Speech-to-Text sample for stream recognize with specified model. To recognize Japanese speech, we have to specify model on recognizing. This is an example code for such international users.
Thanks,

### CODE REVIEW

File: examples/static/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <li><a href="microphone-word-confidence.html">Transcribe from Microphone, with Word Confidence</a></li>
     <li><a href="microphone-streaming-text-to-console.html">Transcribe from Microphone, send text to console</a></li>
     <li><a href="microphone-streaming-object-to-console.html">Transcribe from Microphone, send JSON to console (includes text and metadata)</a></li>
-    <li><a href="microphone-streaming.html">Transcribe from Microphone, Streaming</a></li>
```
Comment (nfriedly) [inline examples/static/index.html]: This should still be there, right?

**Comment by nfriedly** [comment]
Hey, thanks! One comment on the index file, but otherwise this looks good.

**Comment by nfriedly** [comment]
Concept Expansion is listed in the readme in the [ToC](https://github.com/watson-developer-cloud/node-sdk/blob/4bdc68c8056dbb003106fb45fe974bfc0ba4f955/README.md#table-of-contents) and as the example in the [Running in Bluemix](https://github.com/watson-developer-cloud/node-sdk/blob/4bdc68c8056dbb003106fb45fe974bfc0ba4f955/README.md#running-in-bluemix) section

File: examples/static/index.html (hunk 1 above)
Comment (hkomine) [inline examples/static/index.html]: I apologize my late response.
I though that new microphone-streaming-model.html can cover the function of microphone-streaming-model.html.  But we can keep it there as a simple example.

Comment (nfriedly) [inline examples/static/index.html]: Oh, I just realized that there were two entries for micrphone-streaming.html in the list. This makes sense now.

**Comment by nfriedly** [comment]
Thanks again! I think my own mistake confused me, so sorry about that.

File: examples/static/index.html (hunk 1 above)
Comment (hkomine) [inline examples/static/index.html]: Thank you very much for your pulling. Best regards,

**Pull request merged**